app/settings/knockout.go

```diff
@@ -39,6 +40,9 @@ type knockout struct {
 	// Max players shown (excluding danser) on a map. Caps at 50.
 	MaxPlayers int
 
+	// Min players shown (excluding danser) on a map.
```

Comment:
The comment is wrong after your last commit

Reply:
I fixed it after merge